src/base58.h

```diff
@@ -146,7 +146,8 @@ template<typename K, int Size, CChainParams::Base58Type Type> class CBitcoinExtK
 
     K GetKey() {
         K ret;
-        ret.Decode(&vchData[0], &vchData[Size]);
+		if (vchData.size() == 74) //if base58 encouded data not holds a ext key, return a !IsValid() key
```

Comment:
I didn't add a constant because fixing this is already covered in: https://github.com/bitcoin/bitcoin/pull/6215.
The Decode really expects 74 bytes (=hardcode check okay IMO)